server/apps/immich/src/api-v1/asset/asset.service.spec.ts

```diff
@@ -360,6 +373,18 @@ describe('AssetService', () => {
     expect(result).toEqual(assetCount);
   });
 
+  it('get archived asset count by user id', async () => {
+    const assetCount = _getArchivedAssetsCountByUserId();
+
+    assetRepositoryMock.getArchivedAssetCountByUserId.mockImplementation(() =>
+      Promise.resolve<AssetCountByUserIdResponseDto>(assetCount),
+    );
+
+    const result = await sut.getArchivedAssetCountByUserId(authStub.user1);
+
+    expect(result).toEqual(assetCount);
```

Comment:
Not a big deal but you can do this on one line via await expect(...).resolves.toEqual(...)